.changeset/sweet-eels-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```

Comment:
This is not a breaking change according to our versioning policy, as it's only breaking API producers, you could move this to `patch`.

Reply:
Ah ok will change asap.

Reply:
Done